README.md

```diff
@@ -147,6 +147,22 @@ Once the `datadog_agent` module is installed on your `puppetserver`/`puppetmaste
     }
     ```
 
+Installing and pinning specific versions of integrations
+--------------------------------------------------------
+
+    You can specify a given integration and version number to be installed by using `datadog_agent::install_integration`. This will use the `datadog-agent integration` command to ensure a specific integration is installed or uninstalled.
+
+    ```
```

Comment:
We should probably state here how the installation of the integration should happen in the manifests _before_ the config file definition for the integration.

Reply:
Does the config file definition need the integration to be installed in order to generate the config file?

Reply:
No, it's not needed, but since we're not restarting the agent here - we'd be relying on the config file for that, and steps would need to be in order (actually this might not be entirely true if the config files trigger the service restart in a delayed fashion). Alternatively, maybe ideally (though I'm not sure because technically speaking installing an integration does not imply it's enabled), we could notify the service to restart here, maybe in a delayed fashion. And then we could forget any changes to the docs, as they'd be moot.